src/components/vm/disks/diskAdd.jsx

```diff
@@ -417,7 +421,7 @@ export class AddDiskModalBody extends React.Component {
             if (poolTypesNotSupportingVolumeCreation.includes(this.state.storagePoolType)) {
                 validationFailed.storagePool = cockpit.format(_("Pool type $0 does not support volume creation"), this.state.storagePoolType);
             }
-            const poolCapacity = parseFloat(convertToUnit(this.props.storagePools.find(pool => pool.name == this.state.storagePoolName).capacity, units.B, this.state.unit));
+            const poolCapacity = parseFloat(convertToUnit(this.state.storagePools.find(pool => pool.name == this.state.storagePoolName).capacity, units.B, this.state.unit));
```

Comment:
It seems no test caught this, but this looks like a bug. `this.state` has no `storagePools` paremeter. I guess you meant:
```suggestion
            const poolCapacity = parseFloat(convertToUnit(storagePools.find(pool => pool.name == this.state.storagePoolName).capacity, units.B, this.state.unit));
```

Reply:
`this.state.storagePools` is set by componentDidMount, via `get initialState()`.

Reply:
Ah, didn't notice that, thanks